apps/src/templates/progress/ProgressPill.jsx

```diff
@@ -23,6 +23,7 @@ const styles = {
     paddingTop: 6,
     paddingBottom: 6,
     minWidth: 70,
+    lineHeight: '18px',
```

Comment:
Opps - missed adding this file to the commit.
Apparently with lineHeight, you have to use `'18px'` instead of `18` because it's a font size 🤷‍♀️

Reply:
Yes, without units `line-height` is how much to multiply the `font-size` by.